prosody/Dockerfile

```diff
@@ -57,6 +57,7 @@ RUN wget -qO /etc/apt/trusted.gpg.d/prosody.gpg https://prosody.im/files/prosody
     rm -rf /usr/share/lua/{5.1,5.2,5.3} && \
     wget -qO /prosody-plugins/mod_auth_cyrus.lua https://hg.prosody.im/prosody-modules/raw-file/65438e4ba563/mod_auth_cyrus/mod_auth_cyrus.lua && \
     wget -qO /prosody-plugins/sasl_cyrus.lua https://hg.prosody.im/prosody-modules/raw-file/65438e4ba563/mod_auth_cyrus/sasl_cyrus.lua  && \
+    wget -qO /prosody-plugins/mod_http_health.lua https://hg.prosody.im/prosody-modules/raw-file/2b80188448d1/mod_http_health/mod_http_health.lua && \
```

Comment:
This one has a different hash than the rest, not urgent but I'm curious if we can either have the same hash or update the others to a newer one?  Doesn't have to be in this PR.

Reply:
Thanks for merging! :)

Let me try to satisfy your curiosity.. ;)

2b80188448d1/5924 refers to the current tip (June 2024) of the hg repository.
65438e4ba563/4929 refers to a commit from Apr 18 2022.

mod_http_health was added in 5667:9bcd257dea4e (Oct 2023).

That is why 65438e4ba563 was not feasible for mod_http_health.

After diffing 65438e4ba563 and 2b80188448d1 i can confirm, there are no differences in mod_auth_cyrus..

So there is nothing to be said against using the current tip 2b80188448d1 for mod_auth_cyrus too.

Want a PR? :)